test/index.test.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai';
+import { countElements, groupAnagrams } from '../src/index';
+
+describe('countElements', () => {
+    it('passing empty array returns a number', () => {
+        const result = countElements([]);
+        expect(result).to.be.a('number');
+    });
+
+    it('passing a valid input [1, 2, 3] returns expected output', async () => {
```

Comment:
You're declaring a lot of these tests to be `async` but I'm not seeing any `await` keywords. What you're testing doesn't seem to need to be asynchronous so you may be all right if you remove them.

Reply:
Yes, this is copy-paste from somewhere and definitely needs cleanup!